kong/cluster_events/init.lua

```diff
@@ -22,7 +19,7 @@ local CURRENT_AT_KEY         = "cluster_events:at"
 
 
 local MIN_EVENT_TTL_IN_DB = 60 * 60 -- 1 hour
-local PAGE_SIZE           = constants.DEFAULT_CLUSTER_EVENTS_PAGE_SIZE
+local PAGE_SIZE           = 1000
```

Comment:
is this OK to be hardcoded or should it be a variable `db.connector.defaults.pagination.page_size` ?

Reply:
I think we want to have different default for cluster events, so that is authoritative place for that, though a hard-coded one. Not sure how to proceed on that, I think we'll have to wait for feedback from others, perhaps @thibaultcha.